process/economics/economicsData.go

```diff
@@ -378,6 +406,9 @@ func (ed *economicsData) MaxGasLimitPerBlock(shardID uint32) uint64 {
 
 // DeveloperPercentage will return the developer percentage value
 func (ed *economicsData) DeveloperPercentage() float64 {
+	ed.mutRewardsSettings.RLock()
+	defer ed.mutRewardsSettings.RUnlock()
+
```

Comment:
protocolSustainabilityPercentage is not protected by mutex on line 417.

Reply:
done

Reply:
ok